src/dev/i18n/extractors/html.js

```diff
@@ -147,7 +147,33 @@ function trimOneTimeBindingOperator(string) {
   return string;
 }
 
+/**
+ * Remove interpolation expressions from angular and throw on `| i18n:` substring.
```

Comment:
__nit:__ maybe `throw on | i18n: substring` ---> `throw on | i18n: substring within an angular filter ` or something like that?

Reply:
@azasypkin it throws on `| i18n:` substring outside of angular interpolation expression, not within an angular filter.

Reply:
Hmm, not sure I understand, but in any case it's not clear: please add an example in this JSDoc showing what is correct and incorrect usages.